source/educators/concepts/advanced_features/cohorts_overview.rst

```diff
@@ -1,127 +1,73 @@
 .. _Cohorts Overview:
 
-#############################
 Using Cohorts in Your Courses
 #############################
 
 .. tags:: educator, concept
 
-This section provides an introduction to using cohorts to divide the learners
-in your course into smaller groups.
+This section provides an introduction to using cohorts to divide the learners in your course into smaller groups.
```

Comment:
Would it be possible to not change the splitting of the lines? It makes it really difficult to evaluate only the things you have changed.

Reply:
Sure, I will leave them like they were.

Reply:
If you can't revert these changes, could you let me know what you changed in these files? I'm really struggling to understand what you changed, versus what is just altering the paragraph breaks.

Reply:
These documents only needed a few changes. I fixed some titles that didn't meet the RST standards son see also tables and removed one edX reference.

- Cohort Overview: Title standard, correct image code
- Content Experiment Overview : Title standards,edX references in line s 21-31
- Group Configurations: Just title standards.
- Guidelines for Modifying Group Configurations: Title standards, 
-  Offering Different Content to Different Learner Groups: title standars